workspaces/meewee-time-registration/plugins/plugin-meewee-time-registration/meewee-time-registration/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { AuthProvider } from './context/AuthContext';
+import MainComponent from './components/MainComponent/MainComponent';
+
+const App: React.FC = () => {
```

Comment:
`React.FC` is to be avoided: https://backstage.io/docs/architecture-decisions/adrs-adr006

Reply:
Done 👍